src/components/Tooltip/Tooltip.module.scss

```diff
@@ -22,39 +22,6 @@
 }
 
 .popperContainer {
-  .contentBox {
```

Comment:
Having `. popperContainer .contentBox` is probably not needed given this is using CSS Modules, and, I only found the one usage within the tooltip component, so I elected to just move this to the top level and renamed it to `. popperContentBox`. That allows us to actually pass in an override prop `popperBoxStyles` without losing specificity battle.. Personally, I would use as flat as possible for all fo the CSS Modules stuff and the nested SCSS is overall a bit of an anti-pattern with this setup for just this reason (plus the fact it's generally best to keep specificity down to least possible).